src/fastapibootcamp/storage/iers.py

```diff
@@ -0,0 +1,52 @@
+"""Manage IERS the caching for Astropy."""
+
+from __future__ import annotations
+
+from astropy.utils import iers
+from astropy.utils.data import clear_download_cache
+from structlog.stdlib import BoundLogger
+
+
+class IersCacheManager:
+    """Manage IERS the caching for Astropy.
+
+    The Astropy and Astroplan calculations of target observability, as well
+    as the location of the Sun and Moon, use IERS data. To provide good
+    predictions, we configure IERS to cache the IERS tables, but to update
+    them as necessary. This class also allows us to prime the cache with
+    current IERS data. Requests that trigger an IERS download will take longer
+    and will block the app since Astropy downloads IERS data synchronously.
```

Comment:
Technically, another approach would be to write all the handlers and underlying methods that might trigger an IERS download as non-async functions and methods, which will tell FastAPI to run them in a thread pool and avoid blocking the app. I'm not sure if you want to get into that in this comment, though.